src/i18n/i18n_en.json

```diff
@@ -13,5 +13,11 @@
   "header.navigation.about": "About",
   "header.navigation.categories": "Categories",
   "header.userName": "{{props.userName}}",
-  "header.userEmail": "{{props.userEmail}}"
+  "header.userEmail": "{{props.userEmail}}",
```

Comment:
I don't think we need to localize such fields, or I didn't get what you trying to achieve there

Reply:
or its even not your chnages)

Reply:
It is not my change